packages/uiweb/src/lib/components/chat/ChatViewBubble/ChatViewBubble.tsx

```diff
@@ -62,10 +63,10 @@ const SenderMessageProfilePicture = ({ chat }: { chat: IMessagePayload }) => {
   };
   useEffect(() => {
     getUserPfp();
-  }, [account, chat.fromCAIP10]);
+  }, [account, chat.fromDID]);
   return (
     <Section justifyContent="start" alignItems="start">
-      {chat.fromCAIP10.split(':')[1] !== account && (
+      {(chat.fromDID || chat.fromDID).split(':')[1] !== account && (
```

Comment:
`chat.fromDID || chat.fromDID ` aren't these 2 the same? what's the point of OR?

Reply:
yeah, thanks for pointing out will remove one